source/tutorials/write-update-documents.txt

```diff
@@ -101,42 +94,43 @@ To learn more about update operators, see
 Update One Document
 -------------------
 
-To update a single document from a MongoDB collection, call the ``updateOne()``
+To update a single document in a MongoDB collection, call the ``updateOne()``
 method and pass your query filter as the first argument. Then, pass your
-update operators as the second argument to the ``updateOne()`` method. After you call the
-``updateOne()`` method, chain the ``subscribe()`` method and pass a subscriber
-as an argument.
+update operators as the second argument to the ``updateOne()`` method. Finally,
```

Comment:
S: I think this sentence can be condensed and simplified a bit (Github won't let me apply this suggestion to the whole sentence because it's showing diffs in between). Because we show the example and list the parameters in order, I think we can omit stating the order here. We also show the parameters in order in earlier in the page.

```suggestion
To update a single document in a collection, call the ``updateOne()``
method and pass it your query filter and your
update operators. Then,
```